nixos/tests/docker-tools.nix

```
@@ -79,6 +79,16 @@ import ./make-test-python.nix ({ pkgs, ... }: {
             "docker rmi ${examples.nix.imageName}",
         )
 
+    with subtest(
+        "Ensure (layered) nix store has correct permissions "
+        "and that starting un-privileged containers works"
```

```suggestion
        "and that the container starts when its process does not have uid 0"
```

"Priviliged" refers to containers that run with capabilities that could harm the host.